src/analysis/normed_space/star/basic.lean

```diff
@@ -193,3 +193,29 @@ variables {𝕜}
 lemma starₗᵢ_apply {x : E} : starₗᵢ 𝕜 x = star x := rfl
 
 end starₗᵢ
+
+section matrix
+
+local attribute [instance] matrix.normed_group
+
+open_locale matrix
+
+lemma matrix.entrywise_sup_norm_star_eq_norm {n : Type*} [normed_ring E] [star_add_monoid E]
+  [normed_star_monoid E] [fintype n] (M : (matrix n n E)) : ∥star M∥ = ∥M∥ :=
```

Comment:
```suggestion
@[simp] lemma matrix.entrywise_sup_norm_star_eq_norm {n : Type*} [normed_ring E] [star_add_monoid E]
  [normed_star_monoid E] [fintype n] (M : (matrix n n E)) : ∥star M∥ = ∥M∥ :=
```
This seems like a good simp lemma, since the left-hand side is clearly "simpler" than the right-hand side.

Reply:
I agree!  (with left/right switched).  I am not sure how careful I need to be in adding the `@simp` tag, but your suggestion makes me think that it is OK any time there is a clear "simplification" from left to right.